lib/product.rb

```diff
@@ -0,0 +1,29 @@
+class Product
+  attr_reader :category,
+              :name,
+              :unit_price
+
+  def initialize (category, name, unit_price, quantity)
+    @category = category
+    @name = name
+    @unit_price = unit_price
+    @quantity = quantity
+  end
+
+  def quantity
+    @quantity.to_i
+  end
```

Comment:
Nice, I like that you did this -- allows you to maintain how the attribute came in, while still formatting it how you wish it to be returned. good call